packages/codec/Move.toml

```diff
@@ -4,7 +4,7 @@ edition = "2024.beta"
 authors = ["Sui Potatoes"]
 
 [dependencies]
-Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/mainnet" }
+Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "releases/sui-v1.38.0-release" }
```

Comment:
Do we want this change?? (will make depending painful 😆 )

Reply:
It is the version the code is built with and it's universal mainnet / testnet.

Reply:
So no lock-file changes can affect it.